frontend/src/Components/UserProfileInvites.tsx

```diff
@@ -100,6 +100,19 @@ export const UserProfileInvites = observer((props: UserProfileInvitesProps) => {
         }
     };
 
+    const handleEdit = async (code: string, content: string) => {
+        try {
+            const result = await api.inviteAPI.edit(code, content);
+            console.log('EDIT', result);
```

Comment:
It seems here are a couple of lines of debug code left

Reply:
Yeah, same thing with the other handlers. Will clean up in a separate PR.